OJP/OJP_JourneySupport.xsd

```diff
@@ -93,6 +98,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:element ref="OperatorRef" minOccurs="0"/>
+			<xs:element name="OperatorName" type="xs:normalizedString" minOccurs="0">
```

Comment:
Should this be of type `InternationalTextStructure`?

Reply:
updated